git-send-email.perl

```diff
@@ -299,7 +304,7 @@ sub signal_handler {
                     "dump-aliases" => \$dump_aliases);
 usage() unless $rc;
 die "--dump-aliases incompatible with other options\n"
-    if !$help and $dump_aliases and @ARGV;
+    if (!$help) and $dump_aliases and @ARGV;
```

Comment:
On Fri, 10 Jun 2016, Philip Prindeville wrote:

> > @@ -299,7 +304,7 @@ sub signal_handler {
> >                      "dump-aliases" => \$dump_aliases);
> >  usage() unless $rc;
> >  die "--dump-aliases incompatible with other options\n"
> > -    if !$help and $dump_aliases and @ARGV;
> > +    if (!$help) and $dump_aliases and @ARGV;
> 
> The parens around the entire expression are optional.  PBP likes explicit parens because some people can't remember their operator precedence.   I don't agree with all PBP rules... Most Perl operator precedence is fairly obvious (except perhaps for 'next', 'last', etc. being considered operators).

Sure. PBP might recommend this.

All I was saying was that the coding style you introduce is inconsistent
with the existing one. Should you ever choose to submit these patches
upstream, you will be asked to change that.

Don't shoot the messenger.